ui-kit/src/main/kotlin/org/p2p/uikit/atoms/UiKitCounterView.kt

```diff
@@ -0,0 +1,80 @@
+package org.p2p.uikit.atoms
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.Gravity
+import androidx.appcompat.widget.AppCompatTextView
+import androidx.core.content.ContextCompat
+import org.p2p.uikit.R
+import org.p2p.uikit.utils.toPx
+
+private const val VISIBLE_COUNT_MAX_VALUE = 99
+private const val MIN_SIZE_DP = 18
+private const val VERTICAL_PADDING_DP = 0
+private const val HORIZONTAL_PADDING_DP = 4
+private const val PLUS_SIGN = '+'
+
+class UiKitCounterView @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0
+) : AppCompatTextView(context, attrs, defStyleAttr) {
+
+    var count = 0
```

Comment:
I don't really like this patter here. 
I think the components should have only functions.

`setCount`, `updateCount`

The same should be in all other views

Reply:
In Kotlin style we usually call `textView.text =` or `view.graviti =` , `view.isVisible = `, etc. Property style is pretty common